redirects.txt

```diff
@@ -46,6 +46,7 @@ sales/send_quotations/optional_items.rst sales/send_quotations/optional_products
 
 accounting/localizations/nederlands.rst accounting/localizations/netherlands.rst  # (#514)
 
+sales/send_quotations/optional_items.rst sales/send_quotations/optional_products.rst  # (#533)
```

Comment:
Already defined on line 45

Reply:
Thanks.
that must be a mistake I made while rebasing, as this rule isn't mine.
I'll pay attention to my next rebases not to make the same mistake.